addons/website/static/src/js/website.facebook_page.js

```diff
@@ -0,0 +1,18 @@
+odoo.define('website.facebook_page', function (require) {
+
+
+$(document).ready(function() {
+
+    $('.o_facebook_page').each(function() {
```

Comment:
Move this to the snippet animation file and use a snippet animation to implement it.